src/components/ReportWelcomeText.js

```diff
@@ -84,14 +100,19 @@ function ReportWelcomeText(props) {
                 {isChatRoom && (
                     <>
                         <Text>{roomWelcomeMessage.phrase1}</Text>
-                        <Text
-                            style={[styles.textStrong]}
-                            onPress={() => Navigation.navigate(ROUTES.getReportDetailsRoute(props.report.reportID))}
-                            suppressHighlighting
-                        >
-                            {ReportUtils.getReportName(props.report)}
-                        </Text>
-                        <Text>{roomWelcomeMessage.phrase2}</Text>
+                        {/* for rooms in which only admins can post we dont need room name and phrase two */}
+                        {(!isAdminsOnlyPostingRoom || isUserPolicyAdmin) && (
+                            <>
+                                <Text
+                                    style={[styles.textStrong]}
+                                    onPress={() => Navigation.navigate(ROUTES.getReportDetailsRoute(props.report.reportID))}
+                                    suppressHighlighting
+                                >
+                                    {ReportUtils.getReportName(props.report)}
+                                </Text>
+                                <Text>{roomWelcomeMessage.phrase2}</Text>
+                            </>
+                        )}
```

Comment:
I guess the check should have rendered the second phrase if the room was archived

This change caused this regression: https://github.com/Expensify/App/issues/25718